apps/condo/domains/organization/utils/name.utils.js

```diff
@@ -0,0 +1,8 @@
+const SPECIAL_CHARS_REGEXP = /[`!@#$%^&*()_+\-=[\]{};':"\\|,.<>/?~]/
```

Comment:
It's front-end util, so why JS instead of TS?

Reply:
fixed